Sources/Core/AWSSDKHTTPAuth/AWSSigV4Signer.swift

```diff
@@ -127,18 +135,6 @@ public class AWSSigV4Signer: ClientRuntime.Signer {
         )
     }
 
-    public func signEvent(
```

Comment:
The event signing methods on this signer were split off to an extension in the event streams auth module.